modules/runner/src/main/java/org/apache/ignite/runner/internal/app/IgnitionImpl.java

```diff
@@ -0,0 +1,179 @@
+ */
+public class IgnitionImpl implements Ignition {
+    /** */
+    private static final String[] BANNER = new String[] {
+        "",
+        "           #              ___                         __",
+        "         ###             /   |   ____   ____ _ _____ / /_   ___",
+        "     #  #####           / /| |  / __ \\ / __ `// ___// __ \\ / _ \\",
+        "   ###  ######         / ___ | / /_/ // /_/ // /__ / / / // ___/",
+        "  #####  #######      /_/  |_|/ .___/ \\__,_/ \\___//_/ /_/ \\___/",
+        "  #######  ######            /_/",
+        "    ########  ####        ____               _  __           _____",
+        "   #  ########  ##       /  _/____ _ ____   (_)/ /_ ___     |__  /",
+        "  ####  #######  #       / / / __ `// __ \\ / // __// _ \\     /_ <",
+        "   #####  #####        _/ / / /_/ // / / // // /_ / ___/   ___/ /",
+        "     ####  ##         /___/ \\__, //_/ /_//_/ \\__/ \\___/   /____/",
+        "       ##                  /____/\n"
+    };
+
+    /** */
+    private static final String VER_KEY = "version";
+
+    /** */
+    private static final Logger log = LoggerFactory.getLogger(IgnitionImpl.class);
+
+    /** {@inheritDoc} */
+    @Override public synchronized Ignite start(String jsonStrBootstrapCfg) {
+        ackBanner();
+
+        // Vault Component startup.
+        VaultManager vaultMgr = new VaultManager();
+
+        boolean cfgBootstrappedFromPds = vaultMgr.bootstrapped();
+
+        List<RootKey<?, ?>> rootKeys = new ArrayList<>(Collections.singletonList(NetworkConfiguration.KEY));
+
+        List<ConfigurationStorage> configurationStorages =
+            new ArrayList<>(Collections.singletonList(new LocalConfigurationStorage(vaultMgr)));
+
+        // Bootstrap local configuration manager.
+        ConfigurationManager locConfigurationMgr = new ConfigurationManager(rootKeys, configurationStorages);
+
+        if (!cfgBootstrappedFromPds)
+            try {
+                locConfigurationMgr.bootstrap(jsonStrBootstrapCfg);
+            }
+            catch (Exception e) {
+                log.warn("Unable to parse user specific configuration, default configuration will be used", e);
+            }
+        else if (jsonStrBootstrapCfg != null)
+            log.warn("User specific configuration will be ignored, cause vault was bootstrapped with pds configuration");
+
+        NetworkView netConfigurationView =
+            locConfigurationMgr.configurationRegistry().getConfiguration(NetworkConfiguration.KEY).value();
+
+        var serializationRegistry = new MessageSerializationRegistry();
+
+        // Network startup.
+        ClusterService clusterNetSvc = new ScaleCubeClusterServiceFactory().createClusterService(
+            new ClusterLocalConfiguration(
+                "Node" + netConfigurationView.port(),
+                netConfigurationView.port(),
+                Arrays.asList(netConfigurationView.netMembersNames()),
```

Comment:
Get rid of mention the term: network member.

Reply:
Fixed.